source/connect/connection-targets.txt

```diff
@@ -0,0 +1,125 @@
+.. _cpp-connection-targets:
+
+==========================
+Choose a Connection Target
+==========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client, stable api
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use a connection string and ``mongocxx::client`` object
+to connect to different types of MongoDB deployments.
+
+.. include:: /includes/assumes-instance.rst
+
+Atlas
+-----
+
+To connect to a MongoDB deployment on Atlas, include the following elements
+in your connection string:
+
+- URL of your Atlas cluster
```

Comment:
```suggestion
- URI of your Atlas cluster
```

To agree with naming in https://github.com/mongodb/docs-cpp/blob/e22bb3d5bdbd21a5ad2a3dfcf9749a47d039db7d/source/connect/client.txt#L25